app/assets/stylesheets/style.scss

```diff
@@ -490,6 +414,11 @@ label:hover {
 
 .actions .like {
   line-height: 120px;
+  background-color: #83D566;
+}
+
+.actions .like:hover {
+  background-color: #60C13D;
```

Comment:
Color literals like `#60C13D` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#60C13D` should be written as `#60c13d`